edx_django_utils/cache/middleware.py

```diff
@@ -18,16 +18,13 @@ def process_request(self, request):
 
     def process_response(self, request, response):
         """
-         Clear the request cache after processing a response.
-         """
-        RequestCache.clear_all_namespaces()
-        return response
-
-    def process_exception(self, request, exception):    # pylint: disable=W0613
-        """
-        Clear the request cache after a failed request.
+        Clear the request cache after processing a response.
         """
+        # This is to just to do some basic cleanup, and isn't
+        # necessary for correctness; the next request will have the
+        # cache cleared during process_request anyhow.
```

Comment:
[nit] Should we add something like: "Do NOT clear during process_exception, or it will cause issues for later process_response calls that still require the cache." This is non-blocking, so if you think the changelog is good enough as a warning, we can leave as-is.

Reply:
I was thinking that "not required for correctness" would be enough to keep someone who is overly cautious from restoring it, but I could add a note as well.